test/functional/mining_basic.py

```diff
@@ -91,12 +91,6 @@ def assert_submitblock(block, result_str_1, result_str_2=None):
         coinbase_tx.rehash()
 
         # round-trip the encoded bip34 block height commitment
```

Comment:
delete this comment?

Reply:
This hasn't been deleted.

Reply:
Sorry about that :( I'll watch my changes more carefully next time, and look for a chance to take this out in the future.